app/models/import.rb

```diff
@@ -1,6 +1,7 @@
 class Import < ApplicationRecord
   belongs_to :parent, polymorphic: true
   belongs_to :assignee, class_name: "User", optional: true
+  has_many :data_imports, inverse_of: "import"
```

Comment:
Is it possible (and reasonable) to put this on `Imports::Pdf` instead?

Reply:
I had it on `Imports::Pdf` but it broke Administrate's `import#show` page for non-Imports::Pdf, so I ended up moving it back here. I tried adding 

```
def data_imports
  DataImport.none
end
```

but that didn't work either since it wasn't an association.

Reply:
@mike-burns what do you think about 7b22826ba15b7f09b0fb189da2499b1b57d27f91 ?

Reply:
Oooh, cool idea!

I like it a lot. I don't say this often, but would it be helped by a comment above the `-> {none}` explaining what's going on?

Reply:
Added in f1621569b74899a5cad10fe8ff5adb11baa5c00d